src/main/webapp/js/user-page-loader.js

```diff
@@ -39,6 +39,10 @@ function showMessageFormIfLoggedIn() {
       })
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
+          if (loginStatus.username == parameterUsername){
+            const AboutMeClassList = document.getElementById('about-me-form').classList;
```

Comment:
This does not maintain the format of the `messageForm` const below where only the "form" element is saved to a constant. Saving the classList object on the form only seems to make the code more verbose and repetitive.